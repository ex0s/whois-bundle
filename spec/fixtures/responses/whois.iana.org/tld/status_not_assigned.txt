% IANA WHOIS server
% for more information on IANA, visit http://www.iana.org
% This query returned 1 object

domain:       EH

organisation: Not assigned

contact:      administrative
organisation: Not assigned

contact:      technical
organisation: Not assigned

changed:      1999-09-27
source:       IANA

