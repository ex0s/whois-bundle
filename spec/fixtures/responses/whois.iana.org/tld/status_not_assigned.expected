#status
  %s == :available

#available?
  %s == true

#registered?
  %s == false


#created_on
  %s == nil

#updated_on
  %s %CLASS{time}
  %s %TIME{1999-09-27}

#expires_on
  %s %ERROR{AttributeNotSupported}


#registrant_contacts
  %s %CLASS{array}
  %s == []

#admin_contacts
  %s %CLASS{array}
  %s == []

#technical_contacts
  %s %CLASS{array}
  %s == []


#nameservers
  %s %CLASS{array}
  %s == []
